app-office/moneydance/moneydance-2017.8.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+MY_PN="Moneydance"
+
+inherit desktop java-pkg-2 xdg-utils
+
+DESCRIPTION="A personal finance app for Mac, Windows, Linux, iPhone and iPad"
+HOMEPAGE="https://www.moneydance.com/"
+SRC_URI="
+	amd64? ( https://infinitekind.com/stabledl/current/${MY_PN}_linux_amd64.tar.gz -> ${P}-amd64.tar.gz )
+	x86? ( https://infinitekind.com/stabledl/current/${MY_PN}_linux_x86.tar.gz -> ${P}-x86.tar.gz )
+"
+
+LICENSE="Apache-1.0 Apache-2.0 BSD CPAL-1.0 CPL-1.0 CSL-2.0 LGPL-2 MIT TIK"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
```

Comment:
Make sure to add `DEPEND=>=virtual/jdk-1.8` as well.

Reply:
Thanks, added :)